models/census.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var mongoose = require('mongoose');
+
+var censusSchema = mongoose.Schema({
+    'type': String,
+    'activity_year': String,
+    'data': [{
```

Comment:
It's the schema defined in cfpb/hmda-pilot#102. @porterbot can you shed any light on that?